metabase/metabase
# The skeleton of the new homepage

Epic: https://github.com/metabase/metabase/issues/21322

Integration branch for FE homepage changes. 

How to test:
- Open a fresh Metabase install. Make sure you see x-rays for the sample database.
- Add another database. You can add a H2 database and copy the connection string from the sample database. Make sure you can see x-rays for this database and a link to the database on the homepage.
- Create a dashboard and a question. Make sure there recent items on the homepage instead of x-rays.

<img width="920" alt="Screenshot 2022-03-31 at 20 12 48" src="https://user-images.githubusercontent.com/8542534/161299989-4673b24f-3d6b-4b0d-8e31-50526ba94542.png">
<img width="792" alt="Screenshot 2022-03-31 at 20 13 48" src="https://user-images.githubusercontent.com/8542534/161300012-f3ab04f8-7dc5-46e6-ad22-cb538ae42b2d.png">
<img width="1036" alt="Screenshot 2022-04-01 at 19 04 24" src="https://user-images.githubusercontent.com/8542534/161300679-6254e7d8-e255-4ce0-bf6a-fe9dbaf9f531.png">
